Client/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
TIP: You can break the HTML into different sections using comments. This will be easier for the reviewer to review your code and will help you understand what you have written much more.